### PR TITLE
Avoid parsing /etc/{passwd,group} manually

### PR DESCRIPTION
**Pull request by danielkza**
The `Etc` module provides that functionality through standard Unix
system calls. No need to dig through the files.

### CODE REVIEW

File: lib/linux-hub/linux_user.rb [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
     DEFAULT_GROUP = 'linuxhub'
 
-    def self.users_in_group
-      File.open("/etc/group") do |f|
-        f.each_line do |line|
-          user = line.split(":")
-          if user.first == DEFAULT_GROUP
-            return user[3].chomp.split(',').collect { |u| new(username: u) }
-          end
+    class << self
```
Comment (patrobinson) [inline lib/linux-hub/linux_user.rb]: My preference is to use the `self.method` as per the [Github styleguide](https://github.com/styleguide/ruby)

**Comment by patrobinson** [comment]
Just a minor nitpick, also this PR will need a rebase.

But I <3 this change, thank you very much

**Comment by danielkza** [comment]
Rebased and updated.

**Comment by patrobinson** [comment]
Awesome sauce. I'll merge this as is and write some tests before I release a new version, which you've made a lot easier for me!

**Pull request merged**